configs/envoy_front_proxy_v2.template.yaml

```diff
@@ -124,13 +124,6 @@ static_resources:
   {% for service, options in clusters.iteritems() -%}
   - {{ helper.internal_cluster_definition(service, options)|indent(2) }}
   {% endfor %}
-dynamic_resources:
```

Comment:
Did this config not even use SDS? I guess not? I'm confused why this was even in here?